stage/configuration/test_directory_origin.py

```diff
@@ -633,11 +633,74 @@ def test_directory_origin_configuration_include_custom_delimiter(sdc_builder, sd
 
 
 @pytest.mark.parametrize('data_format', ['XML'])
-@pytest.mark.parametrize('include_field_xpaths', [False, True])
-@pytest.mark.skip('Not yet implemented')
-def test_directory_origin_configuration_include_field_xpaths(sdc_builder, sdc_executor,
+@pytest.mark.parametrize('include_field_xpaths', [True, False])
+def test_directory_origin_configuration_include_field_xpaths(sdc_builder, sdc_executor, shell_executor, file_writer,
                                                              data_format, include_field_xpaths):
-    pass
+    """Test for Directory origin can read XML file with include field xpath parameter as true or false.
+    Here we will be creating XML file with namespaces .
+
+    Include Filed Xpaths |Expected outcome
```

Comment:
Probably a typo `Field Xpaths`
```suggestion
    Include Field Xpaths |Expected outcome
```